src/core/lib/channel/channel_stack_builder.h

```diff
@@ -63,6 +64,7 @@ class ChannelStackBuilder {
   grpc_transport* transport() const { return transport_; }
 
   // Set channel args.
+  // args *must be* preconditioned already.
   ChannelStackBuilder& SetChannelArgs(const ChannelArgs& args);
```

Comment:
Do we still call this anywhere?  If not, we can remove it.

Reply:
It was used in one place, but I was able to refactor that usage away without much extra spaghettification.